frontend/components/product-list/ProductListView.tsx

```diff
@@ -39,7 +40,9 @@ export function ProductListView({
                   px={{ base: 3, sm: 0 }}
                >
                   <ProductListBreadcrumb productList={productList} />
-                  <ProductListDeviceNavigation productList={productList} />
+                  {productList.type !== ProductListType.AllTools && (
+                     <ProductListDeviceNavigation productList={productList} />
+                  )}
```

Comment:
Looks like there's similar logic for hiding this inside the ProductListDeviceNavigation component. Let's move this logic there and keep it all in one place.

Reply:
Resolved c22ba9d048917f11442ac587815d5f30abd4db28